switch:
    ## Registration, Colorization and Mapping
    # view
    kCreateDensePcd: false
    kViewRegistration: false
    kFullViewMapping: false

    ## Data Process
    kLidarFlatProcess: true
    kFisheyeFlatProcess: true

    ## Calibration and Optimization cost analysis
    kCeresOptimization: true
    kMultiSpotsOptimization: true
    kAnalysis: false
    
    # spot
    kSpotRegistration: false
    kGlobalMapping: true

    # colorization
    kSpotColorization: false
    kGlobalColoredMapping: false

    kGlobalUniformSampling: false

spot:
    kOneSpot: -1  # -1: means run all the spots, other means run the specific spot index 

essential:
    kLidarTopic: "/livox/lidar"
    # kDatasetName: "crf"
    # kNumSpots: 1
    # kDatasetName: "rb1"
    # kNumSpots: 1
    kDatasetName: "bs_hall"
    kNumSpots: 1
    kNumViews: 5
    kImageRows: 2048
    kImageCols: 2448
    kAngleInit: -50
    kAngleStep: 25

transform:
    rx: -0.0
    ry: 3.13
    rz: -0.861748
    tx: -13.21285
    ty: -23.67988
    tz: -0.310397
